core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdmin.java

```diff
@@ -25,14 +24,6 @@ public class ConsensusCommitAdmin implements DistributedTransactionAdmin {
   private final String coordinatorNamespace;
   private final boolean isIncludeMetadataEnabled;
 
-  @Inject
-  public ConsensusCommitAdmin(DistributedStorageAdmin admin, DatabaseConfig databaseConfig) {
-    this.admin = admin;
-    ConsensusCommitConfig config = new ConsensusCommitConfig(databaseConfig);
-    coordinatorNamespace = config.getCoordinatorNamespace().orElse(Coordinator.NAMESPACE);
-    isIncludeMetadataEnabled = config.isIncludeMetadataEnabled();
-  }
```

Comment:
I don't think think we should remove this constructor because it's used by the Guice module:
https://github.com/scalar-labs/scalardb/blob/733e167368f18c50c5790691c56bd63c9b57d221/core/src/main/java/com/scalar/db/service/TransactionModule.java

Reply:
Right, thank you for catching this. 
I reverted the deletion, please have a look again.